components/localize/rtl-mixin.js

```diff
@@ -0,0 +1,14 @@
+export const RtlMixin = superclass => class extends superclass {
```

Comment:
Not sure if this is the correct place for this mixin... should it go into an `i18n` or `internationalization` folder?

Reply:
Yeah... and neither of these are really "components" either eh. We may also want to pull some of these things we foresee being very common into a single file for export like Lit does with `import { html, css, whatever } from 'lit-element/lit-element.js';`.

Reply:
Yeah, that could be nice.